### PR TITLE
clean up UI source code

### PR DESCRIPTION
**Pull request by sappharx**
breaks down big components into smaller, stateless functional components, as possible

### CODE REVIEW

File: src/app/leaderboard/user.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <div className={ classes(user, me, canChallengeOpponent) } onClick={ clickHandler } >
+      <div>
+        <span>{index + 1}</span>
+        <Avatar image={ user.image } />
+        <UserName user={ user } />
+        <Delta user={ user } userDelta={ deltas.find((delta) => delta.userID === userID) } />
+        <Rating user={ user } />
+        <Record user={ user } />
+        {/*
```
Comment (Keysox) [inline src/app/leaderboard/user.jsx:31]: We may want to comment out the braces here and down below

Comment (sappharx) [reply src/app/leaderboard/user.jsx:31]: can't; it's invalid jsx syntax

**Comment by Keysox** [comment]
:shipit: cc @zenblender

File: src/app/leaderboard/user.jsx (hunk 1 above)
Comment (sappharx) [reply src/app/leaderboard/user.jsx:31]: maybe we could use html style comments, but I don't think it's worth it

**Comment by zenblender** [comment]
Are we reviving work on this?  It was a fun project!  :shipit:

**Pull request merged**